gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -7,28 +7,32 @@
 import React from 'react';
 import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import { NxP, NxCode } from '@sonatype/react-shared-components';
 
 import NxPageTitlePolicyViolationIndicatorExample from './NxPageTitlePolicyViolationIndicatorExample';
+import NxPageTitleEverythingExample from './NxPageTitleEverythingExample';
 
 const nxPageTitleCode = require('./NxPageTitleExample.html'),
     nxPageTitleActionsCode = require('./NxPageTitleActionsExample.html'),
+    nxPageTitleSubtitleCode = require('./NxPageTitleSubtitleExample.html'),
     nxPageTitlePolicyViolationIndicatorCode =
-      require('./NxPageTitlePolicyViolationIndicatorExample.tsx?raw');
+      require('./NxPageTitlePolicyViolationIndicatorExample.tsx?raw'),
+    nxPageTitleEverythingCode = require('./NxPageTitleEverythingExample.tsx?raw');
 
 const NxPageTitlePage = () =>
   <>
     <GalleryDescriptionTile>
-      <p className="nx-p">
+      <NxP>
         <code className="nx-code">nx-page-title</code> is used at the top of a page, it always has a title, and can also
-        have an icon and descriptive text.
-      </p>
-      <p className="nx-p">
-        In addition <code className="nx-code">.nx-page-title</code> can have tags or actions inline with the title.
-      </p>
-      <p className="nx-p">
+        have a sub-title, descriptive text, and space for tags or other indicators.
+      </NxP>
+      <NxP>
+        In addition <code className="nx-code">.nx-page-title</code> can have buttons inline with the title.
```

Comment:
Do we want to document more clearly how to do the buttons (e.g. adding a row to the classes table for `.nx-btn-bar`) or is documenting by example enough?

Reply:
> Do we want to document more clearly how to do the buttons (e.g. adding a row to the classes table for `.nx-btn-bar`) or is documenting by example enough?

I think that's worth documenting: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/ca7fdcb455dcc36e625599b5665502e0cf3152c5